hedera-node/src/test/java/com/hedera/services/contracts/execution/CallEvmTxProcessorTest.java

```diff
@@ -103,12 +107,14 @@ class CallEvmTxProcessorTest {
     private final long INTRINSIC_GAS_COST = 290_000L;
     private final long GAS_LIMIT = 300_000L;
 
+    private FakeHederaTracer fakeHederaTracer;
```

Comment:
Good use of fakes instead of mocks.  I approve.

Reply:
Actually, it's been removed now as we reverted back to using an inline-created tracer instead of a dagger managed one 😅